.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
+x@name: CI
```

Comment:
You broke CI completely here by accident.

Reply:
Fixed.
Accidentally edited the file.